### PR TITLE
Build the body and the footer

### PR DESCRIPTION
**Pull request by Grandi0z**
#### In this code we :

- Give a column structure to the project with Grid and Flexbox
- Add background
- stylize the text content

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,5 +35,156 @@ <h1 class="me">Hey there. I’m Mariane
...
+            </article>
+            <article class="box">
+                <section class="in-box">
+                    <h3 class="">Multi-Post Stories Gain+Glory</h3>
+                    <div class="btn-language 1 ">
+                        <button type="button">Ruby on rails</button>
+                        <button type="button">css</button>
+                        <button type="button">JavScript</button>
+                        <button type="button">html</button>
+                    </div>
```
Comment (emmiiorji) [inline index.html]: You have a beautiful design. However, you'll agree that the list of project technologies is just that - a list. I think that using an `li` tag for each item and a `ul` tag to enclose them all will be more semantic.

Remember to modify it in all the projects.

File: style/styles.css [hunk 2]
```diff
@@ -77,4 +177,139 @@ header > a :hover {
...
+  align-items: center;
+  gap: 16px;
+}
+
+#recent-works {
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  height: auto;
+}
```
Comment (emmiiorji) [inline style/styles.css]: As mentioned in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/100a933087f033def9233c8cce29c72194391fe0/portfolio/2_mobile.md?plain=1#L51), kindly use a `display: grid` property for the container of the works section.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Grandi0z, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more


## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adherence to GitHub flow
- Good design


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Grandi0z 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**